sites/public/pages/listings.tsx

```diff
@@ -272,6 +273,12 @@ const ListingsPage = () => {
               controlClassName="control"
               options={communityTypeOptions}
             />
+            <FieldGroup
+              type="checkbox"
+              name="include_nulls"
```

Comment:
Can we name this similar to the other wired-up filters, using the key as the name, like
```
{FrontendListingFilterStateKeys.includeNulls}
```
That way we're sure we're using the same string everywhere, so we won't get bugs from typos.